lib/mount.c

```diff
@@ -8,6 +8,11 @@
   See the file COPYING.LIB.
 */
 
+/* For environ */
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
```

Comment:
Hmm, not sure this actually worked.

Locally my tests pass, but since I defined `_GNU_SOURCE` here, I've been getting test failures on github.